pkg/visor/init.go

```diff
@@ -84,6 +95,8 @@ var (
 	pvs vinit.Module
 	// Public visor: advertise current visor as public
 	pv vinit.Module
+	// Transport check module
```

Comment:
interesting idea to implement non-required dependencies as a regular module that is not included in visor but is initialized too, didn't think of it.
I would add a comment here that this is not a functional module but a grouping of all transport types initializations

Reply:
Okay, will do.